### ISSUE TITLE
Horizontal Scroll-ability to allow long labels

### ISSUE THREAD

**Issue by djsegal** [offirgolan/ember-light-table#135]
I dont know how to fix long labels being too crowded
- Is there already a way to add horizontal scrollability? 
- I couldn't find an example like this in the demo

---

<img width="510" alt="screen shot 2016-08-03 at 12 13 17 pm" src="https://cloud.githubusercontent.com/assets/3156114/17372894/c7d77d6a-5973-11e6-97e1-74334f0a3d14.png">

**Comment by jamesdixon** [comment]
@djsegal typically, you could do something like this:

```html
<div style="overflow-x: auto">
  <table>...</table>
</div>
```
However, this doesn't appear to work. It may be because of the use of `ember-scrollable`. 

Would love to hear from the collaborators on this as well.

P.S. may be related to #131

**Comment by taras** [comment]
@jamesdixon I'll take a look at this soon as I can. It'll likely be about 2 weeks. How big of an issue is this for you right now?

**Comment by jamesdixon** [comment]
@taras not a big issue at this point. I know how busy you are, so whenever you're able to look at it, it's very much appreciated!

**Comment by taras** [comment]
@jamesdixon I have a client who just pinged me about this. I told them that I'll look at it in 8 days. So I'll be back :)

**Comment by HenryVonfire** [comment]
Until an official solution is provided, something that worked for me was to set each column with a fixed width and then wrapping the table with two `div`s like this:

```
<div style="overflow:auto">
  <div style="width: total_width_columns">
    <table>...</table>
  </div>
</div>
```

If you want it to work with the resize option too, you can bind the width with a computed property that recalculates the total width after a resizing.

**Comment by jamesdixon** [comment]
@HenryVonfire very nice! I'll give this a shot

**Comment by bargar** [comment]
@HenryVonfire thanks for the solution!

Refinement: use `min-width: total_width_columns` instead of `width` and you get:

* scrollbar when needed (i.e. when available space is below that defined in the total of the column widths)
* maximal use of horizontal space when available (e-l-t expands columns beyond their specified column width, space permitting)

In this way, we don't end up with an awkwardly narrow table in larger screens because we are defensively designing for smaller ones.

CC @jamesdixon

**Comment by djsegal** [comment]
@jamesdixon, so are we making @bargar's solution the official one?

it seems a little too roll-your-owny

**Comment by offirgolan** [comment]
@djsegal I'm really sorry for the delay on this. @taras and I have been a bit preoccupied by some other high priority tasks. We hope to have a solution to this sometime in the near future.

**Comment by offirgolan** [comment]
@djsegal #222 solves this use-case. That PR is a rudimentary solution that is largely a play on what @bargar suggested. I dont mind merging it in as a temporary solution but I want to do so with confidence. Feel free to test it out and let me know.

**Comment by offirgolan** [comment]
Resolved in #222.
